.github/workflows/porter-integration-pr.yml

```diff
@@ -124,3 +124,33 @@ jobs:
         PORTER_INTEG_FILE: uninstall_test.go
       run: go run mage.go -v TestIntegration
       shell: bash
+  signing_test_integ:
```

Comment:
Should we make another reusable workflow file?

Reply:
Didn't even think of that, it would be a great idea. I will update the PR

Reply:
Unfortunately it is not possible. The documentation states that it is a maximum of 20 reusable workflows from the top-level workflow file 😢
https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations